src/service/viewport-impl.js

```diff
@@ -191,6 +191,17 @@ export class Viewport {
   }
 
   /**
+   * Sets the body border-bottom to the specified value.
+   * @param {number} borderBottom
+   */
+  setBorderBottom(borderBottom){
+    onDocumentReady(this.win_.document, () => {
```

Comment:
so this function is both async/sync? feels icky.

couldnt we at least use `whenDocumentReady` and do something like

```js
updateBorderBottom(val) {
  return this.docReadyPromise.then(() => {
    //.. your update code here
  });
}
```